### PR TITLE
Create laravel.yml

### PR DESCRIPTION
**Pull request by xubom123**

### CODE REVIEW

**Comment by BrandonXLF** [comment]
This project does not use Laravel

**Review by xubom123 — COMMENTED** [review]
Help

File: .github/workflows/laravel.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    steps:
+    - uses: shivammathur/setup-php@15c43e89cdef867065b0213be354c2841860869e
+      with:
+        php-version: '8.0'
+    - uses: actions/checkout@v4
+    - name: Copy .env
+      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist
```
Comment (xubom123) [inline .github/workflows/laravel.yml:22]: Run

**Review by xubom123 — COMMENTED** [review]
Ten

**Comment by crazyguero420** [comment]
****

**Pull request closed without merging**